radio/src/gui/colorlcd/model_logical_switches.cpp

```diff
@@ -162,6 +162,13 @@ class LogicalSwitchEditPage: public Page
                            cs->v1 = newValue;
                            SET_DIRTY();
                            v2Edit->invalidate();
```

Comment:
I believe this line should be inside the `if` block, at the end of it. It triggers a re-draw of the UI component. Functionally, it does not make a difference (whereby, it makes sense to check it for `nullptr` value before dereferencing), but logically, it is cleaner.

Reply:
Ah, yes... that makes sense... I was sort of wondering why it could get away with that as everything I added was protected within the nullptr check ;). Moved.